generic/tests/ping.py

```diff
@@ -32,10 +36,11 @@ def _get_loss_ratio(output):
 
     timeout = int(params.get("login_timeout", 360))
     ping_ext_host = params.get("ping_ext_host", "no") == "yes"
+    ping_default = params.get("ping_default", "no") == "yes"
 
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
-    error.context("Login to guest", logging.info)
+    logging.info("Login to guest")
```

Comment:
It is suggested to use error_context.context("xxx", logging.info) for test steps prompts.